### ISSUE TITLE
Error in SMS Notification module

### ISSUE THREAD

**Issue by ringe** [freescout-helpdesk/freescout#2295]
I can't get the SMS Notification module to work:

```
Undefined offset: 3 {"userId":7,"email":"private@example.com","exception":"[object] (ErrorException(code: 0): Undefined offset: 3 at /var/www/example.com/public_html/Modules/SmsNotifications/Providers/SmsNotificationsServiceProvider.php:158)
[stacktrace]
#0 /var/www/example.com/public_html/Modules/SmsNotifications/Providers/SmsNotificationsServiceProvider.php(158): Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleError()
#1 [internal function]: Modules\\SmsNotifications\\Providers\\SmsNotificationsServiceProvider->Modules\\SmsNotifications\\Providers\\{closure}()
#2 /var/www/example.com/public_html/overrides/tormjens/eventy/src/Action.php(27): call_user_func_array()
#3 /var/www/example.com/public_html/overrides/laravel/framework/src/Illuminate/Support/Collection.php(339): TorMorten\\Eventy\\Action->TorMorten\\Eventy\\{closure}()
#4 /var/www/example.com/public_html/overrides/tormjens/eventy/src/Action.php(28): Illuminate\\Support\\Collection->each()
#5 /var/www/example.com/public_html/vendor/tormjens/eventy/src/Events.php(139): TorMorten\\Eventy\\Action->fire()
#6 /var/www/example.com/public_html/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(221): TorMorten\\Eventy\\Events->action()
#7 /var/www/example.com/public_html/app/Subscription.php(406): Illuminate\\Support\\Facades\\Facade::__callStatic()
#8 /var/www/example.com/public_html/app/Http/Middleware/TerminateHandler.php(18): App\\Subscription::processEvents()
#9 /var/www/example.com/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(218): App\\Http\\Middleware\\TerminateHandler->terminate()
#10 /var/www/example.com/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(189): Illuminate\\Foundation\\Http\\Kernel->terminateMiddleware()
#11 /var/www/example.com/public_html/public/index.php(88): Illuminate\\Foundation\\Http\\Kernel->terminate()
#12 {main}
"}
                    
```

I'm not able to to see what array is missing something:
```
        // Send notifications.
        \Eventy::addAction('subscription.process_events', function($notify) {
            if (empty($notify[self::MEDIUM_SMS])) {
                return;
            }
            $delay = now()->addSeconds(Conversation::UNDO_TIMOUT);
            foreach ($notify[Subscription::MEDIUM_MOBILE] as $conversation_id => $notify_info) {
                \Modules\SmsNotifications\Jobs\SendSmsNotificationToUsers::dispatch($notify_info['users'], $notify_info['conversation'], $notify_info['threads'])
                    ->delay($delay)
                    ->onQueue('default');
            }
        });
```

**Comment by freescout-helpdesk** [comment]
Fixed in SMS Notifications Module v1.0.1

**Comment by ringe** [comment]
Well done! :)
